README.md

```diff
@@ -240,6 +240,48 @@ const render = () => html`
 `;
 ```
 
+#### `asyncAppend(asyncIterable)` and `asyncReplace(asyncIterable)`
+
+`asyncAppend` renders the values of an [async iterable](https://github.com/tc39/proposal-async-iteration),
+appending each new value after the previous.
+
+`asyncReplace` renders the values of an [async iterable](https://github.com/tc39/proposal-async-iteration),
+replacing the previous value with the new value.
+
+Example:
+
+```javascript
+/**
+ * Returns an async iterable that yields the number of seconds since `date`,
+ * approximately every second.
+ */
+async function* secondsAgo(date) {
+  const seconds = (Date.now() - date.getTime()) / 1000;
```

Comment:
Perhaps there is a simpler form example? I find datetime in this example  somewhat distracting. Perhaps setTimeout(, 1000) is sufficient?

Reply:
Made a much simpler `countUp()` example :)